parameter_files/fates_params_default.cdl

```diff
@@ -434,12 +434,12 @@ variables:
 	double fates_phen_evergreen(fates_pft) ;
 		fates_phen_evergreen:units = "logical flag" ;
 		fates_phen_evergreen:long_name = "Binary flag for evergreen leaf habit" ;
+	double fates_phen_fnrt_drop_fraction(fates_pft) ;
+		fates_phen_fnrt_drop_fraction:units = "fraction" ;
+		fates_phen_fnrt_drop_fraction:long_name = "fraction of fine roots to drop during drought/cold" ;
```

Comment:
@rgknox Quick follow-up to our Zoom discussion yesterday on parameter names. 

If you think this is fine, I would rather keep the new name `fates_phen_fnrt_drop_fraction` instead of `fates_phen_fnrt_drop_frac` just so the parameter names are consistent with similarly named variables (`fates_phen_stem_drop_fraction` and `fates_phen_flush_fraction`).

Reply:
geeze, I see that we did not maintain this name change request, sorry about that.  We can check-in about this.